pos_partner_deselection/__manifest__.py

```diff
@@ -2,8 +2,8 @@
 # Copyright 2019 Kolushov Alexandr <https://it-projects.info/team/KolushovAlexandr>
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 {
-    "name": """Deselect Partner""",
-    "summary": """POS Partner deselection after a defined time interval""",
+    "name": """Partner deselection""",
```

Comment:
Name is not changed. Capital letter for the second word

Reply:
fixed